tracer/test/test-applications/integrations/Samples.Kafka/Config.cs

```diff
@@ -6,7 +6,7 @@ namespace Samples.Kafka
 {
     internal static class Config
     {
-        public static readonly string KafkaBrokerHost = Environment.GetEnvironmentVariable("KAFKA_BROKER_HOST") ?? "localhost:9092";
+        public static readonly string KafkaBrokerHost = Environment.GetEnvironmentVariable("KAFKA_BROKER_HOST") ?? "localhost:9093";
```

Comment:
Why does this need to change? 🤔

Reply:
I used my own deployment of kafka to run tests locally, forgot to change the value back. Thanks for noticing!